client/src/containers/AddressPage.js

```diff
@@ -139,84 +159,154 @@ export default class AddressPage extends Component {
               userAddr={this.state.searchAddress}
               numOfAssocAddrs={this.state.assocAddrs.length}
             />
-            { this.state.userAddr &&
+            {this.state.userAddr && (
               <div className="float-left">
                 <h5 className="primary">
-                  <Trans>PORTFOLIO: Your search address is associated with <u>{this.state.assocAddrs.length}</u> <Plural value={this.state.assocAddrs.length} one="building" other="buildings" />:</Trans>
+                  <Trans>
+                    PORTFOLIO: Your search address is associated with{" "}
+                    <u>{this.state.assocAddrs.length}</u>{" "}
+                    <Plural
+                      value={this.state.assocAddrs.length}
+                      one="building"
+                      other="buildings"
+                    />
+                  </Trans>
+                  :
                 </h5>
                 <ul className="tab tab-block">
-                  <li className={`tab-item ${this.state.currentTab === 0 ? "active" : ""}`}>
-                    <a // eslint-disable-line jsx-a11y/anchor-is-valid
+                  <li
+                    className={`tab-item ${
+                      this.props.currentTab === 0 ? "active" : ""
+                    }`}
+                  >
+                    <Link
+                      to={this.generateBaseUrl()}
+                      tabIndex={this.props.currentTab === 0 ? -1 : 0}
```

Comment:
I added this `tabIndex` prop here so that folks tabbing through the tabs (hah!) won't be prompted to click on the tab they are currently viewing. 

I also acknowledge that this isn't the most DRY solution in checking our `currentTab` prop, but since there are so many more severly non-DRY implementations throughout this component, I thought we should save refactoring for when we do a full sweep of the codebase.